packages/client/hmi-client/src/model-representation/petrinet/petrinet-renderer.ts

```diff
@@ -167,6 +168,20 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 			.style('pointer-events', 'none')
 			.html((d) => d.label);
 
+		// transitions expression text
+		transitions
+			.append('text')
+			.attr('y', (d) => -d.height / 2 - 5)
+			.style('text-anchor', 'middle')
+			.style('paint-order', 'stroke')
+			.style('stroke', (d) =>
+				d.data.strataType ? strataTypeColors[strataTypes.indexOf(d.data.strataType)] : '#FFF'
+			)
+			.style('stroke-width', '3px')
+			.style('stroke-linecap', 'butt')
+			.style('fill', 'var(--text-color-primary')
+			.style('pointer-events', 'none')
+			.html((d) => d.data.expression ?? null);
```

Comment:
- Why do you care about strataTypes?

Reply:
@mwdchang copy pasted this from the previous label text, its not needed.  I've removed this and given the default which was '#FFF'